tests/fixtures/rules/require-spread/good/super.js

```diff
@@ -1,4 +1,4 @@
-let Blah = Ember.Object.extend({
+let Blah = Object.extend({
```

Comment:
Can you change `Object` to `Foo`? Even though this has no impact on the tests, I'd rather not make this look like we're extending from the base JS `Object` function.